cluster-api/cloud/google/machineactuator.go

```diff
@@ -104,6 +106,14 @@ func NewMachineActuator(kubeadmToken string, machineClient client.MachineInterfa
 		}
 	}
 
+	var configWatch *machinesetup.ConfigWatch
+	if configListPath != "" {
```

Comment:
I don't think this check is necessary. `machinesetup.NewConfigWatch` should return an error if this is not a valid path to a file and this function should error-out approrpiately.

Reply:
Create is the only command that has to pass a config list path, and then add and delete pass an empty string to the method. I put this check here to handle the empty string from add and delete, and create has a check to prevent an empty string. NewConfigWatch() checks that the path is valid. I can move the empty string check to NewConfigWatch() so it just returns nil (but no error) if it gets an empty string if that makes more sense.

Reply:
So would add/delete also need to pass the config path when we switch over (and I should add TODOs there too)?